apps/balancer-tools/src/components/ui/button.tsx

```diff
@@ -0,0 +1,56 @@
+import { Slot } from "@radix-ui/react-slot";
```

Comment:
👍 cool implementation, but can you merge this into our already existing `Button` component?

Reply:
Hm, didn't realize it was in the `ui` folder, I might move them to components